test/check-application

```diff
@@ -1131,6 +1193,58 @@ class TestFiles(testlib.MachineCase):
         b.click("button.pf-m-link")
         b.wait_not_present(".pf-v5-c-modal-box")
 
+        # Test permissions in details view
+        b.mouse("[data-item='home']", "dblclick")
+        b.mouse("[data-item='admin']", "dblclick")
+        b.click("button[aria-label='Display as a list']")
+        basedir = "/home/admin"
+        for i in range(8):
+            m.execute(f"runuser -u admin touch {basedir}/file{i}")
+
+        def check_perms_match(file: str, basedir: str) -> None:
+            ls = m.execute(f"ls -l {basedir}/{file}")[:10]
+            ui = b.text(f"[data-item='{file}'] td:nth-child(4) pre").replace(" ", "")
+            self.assertEqual(ls[1:], ui)
+
+        # Simple permissions
+        m.execute(f"chmod 000 {basedir}/file0")
+        m.execute(f"chmod 111 {basedir}/file1")
+        m.execute(f"chmod 222 {basedir}/file2")
+        m.execute(f"chmod 333 {basedir}/file3")
+        m.execute(f"chmod 444 {basedir}/file4")
+        m.execute(f"chmod 555 {basedir}/file5")
+        m.execute(f"chmod 666 {basedir}/file6")
+        m.execute(f"chmod 777 {basedir}/file7")
```

Comment:
Small nitpick, I'd just go for one `m.execute()` not sure how expensive multiple's are but I feel that they are not free.

we can just do a multiline string:

```
m.execute(f"""
""")
```